common.js

```diff
@@ -8,13 +9,33 @@ const VERSIONS_JSON = 'https://ziglang.org/download/index.json';
 const MACH_VERSIONS_JSON = 'https://pkg.machengine.org/zig/index.json';
 const CACHE_PREFIX = "setup-zig-global-cache-";
 
+const MINIMUM_ZIG_VERSION_REGEX = /\.minimum_zig_version\s*=\s*"(.*?)"/;
```

Comment:
```suggestion
const MINIMUM_ZIG_VERSION_REGEX = /\.\s*minimum_zig_version\s*=\s*"(.*?)"/;
```
My bad, I forgot to allow whitespace here too in the regex I sent you.
This can do weird things on some specific ill-formed ZON, but I'm not too worried about that.